### DISCUSSION TITLE
Changing the license type.

### DISCUSSION THREAD

**Discussion by pfmephisto** [linkarkitektur/SpeckleLCA#30] [Polls]
In our lates developer call the topic of changing the license was brought up. The argument, was that a more permissive license would make it more a attractive to companies to integrate with the project. There for the proposal was to change the license to maybe MIT or Apache2.
The intent with the original license was to ensure that anyone's contribution would keep benefiting the community. It should still have been possible for anyone/companies to use the code commercially but that improvements to the actual project would flow back up stream. 

With the current development of the project, it looks like it might end up more as a software library who finds it self implemented in a web app but could also be used separately. Therefore It might make more sense to change the license to LGPL. This would require all changes to the library to be made available open source, but anyone would still be able to use the library within their closed sourced software. 

The argument for using MIT or Apache2 would be that realistically no body would have the resources to enforce the licenses anyway and we therefore might just forgo the extra complexity and choose a permissive license.

This poll is intended to here the communitys opinion and have a discussion on the subject.

**Comment by rhughes42** [comment]
@pfmephisto closing discussion based on poll results. Assumed license is updated @fabianlinkflink.

**Reply by pfmephisto** [reply]
Actually since the refactor it's not in anymore. 
We would need to add a new license file.
